spec/spec_helper.rb

```diff
@@ -18,3 +18,6 @@
 # Turn off all the OmniAuth logging messages
 # This can be helpful for debugging, in which case just comment out this line
 OmniAuth.config.logger = Logger.new('/dev/null')
+
+# Override OmniAuth 2.0 default to only support POST requests since the smart workflow uses GET
+OmniAuth.config.allowed_request_methods = [:get, :post]
```

Comment:
@Ortuna do we need this change with your updates?

Reply:
It shouldn't be an issue having it in both places.